TraceScripts/README.md

```diff
@@ -60,4 +60,36 @@ The numeric values of the "logAreaFlags_ApiInOut", "logAreaFlags_FnInOut", and "
 2. Enable **FileSharing** for your application, this can be done through the **Info.plist** file of your application.
 3. Copy the `PlayFabMultiplayerLogger.json` file into the `~/Documents` directory.
 4. Run the application.
-5. Locate the output log inside the directory specified by the "logFolder" property of `PlayFabMultiplayerLogger.json`.
\ No newline at end of file
+5. Locate the output log inside the directory specified by the "logFolder" property of `PlayFabMultiplayerLogger.json`.
+
+### Android: Instructions
+
+> **Note: Gathering PlayFabMultiplayer Logs on Android only works for Android API 29 and below, due to Google's app-scoped storage permission changes for Android API 30 and beyond. This will be fixed in a future release.**
+
+> *Note: Relevant documentation for copying files to and from Android devices can be found [here](https://developer.android.com/tools/adb#copyfiles).*
```

Comment:
is there a reason the first Note appears bolded and the second italicized?

Reply:
I wanted to convey levels of importance in the notes - the second note isn't super crucial, it's just additional details, whereas the first note is very crucial.

Reply:
gotcha. I think I see what you mean. I'm fine deferring to your preference here and don't feel strongly but think the two notes stacked on top of each other like that look a little funky. Ultimately not the end of the world though :)

Reply:
Yeah, I see what you mean...I think I'll keep it unbolded, but will remove the italicization.